webapp/components/admin_console/webhook_settings.jsx

```diff
@@ -124,7 +124,7 @@ export default class WebhookSettings extends AdminSettings {
                     label={
                         <FormattedMessage
                             id='admin.service.overrideTitle'
-                            defaultMessage='Enable Overriding Usernames from Webhooks and Slash Commands: '
+                            defaultMessage='Enable Integrations to Override Usernames: '
```

Comment:
Propose "Enable webhooks and slash commands to override usernames"? 

We may have other integrations to which this feature is not relevant.